test/testsets.json

```diff
@@ -31,40 +29,42 @@
     { "name": "test_fs_stat.js", "skip": ["nuttx"], "reason": "not implemented for nuttx" },
     { "name": "test_fs_writefile_unlink.js" },
     { "name": "test_fs_writefile_unlink_sync.js" },
-    { "name": "test_fs.js", "skip": ["nuttx"], "reason": "not implemented for nuttx" },
-    { "name": "test_fs1.js", "skip": ["nuttx"], "reason": "not implemented for nuttx" },
-    { "name": "test_fs2.js", "skip": ["nuttx"], "reason": "not implemented for nuttx" },
-    { "name": "test_gpio1.js", "skip": ["all"], "reason": "needs hardware" },
-    { "name": "test_gpio2.js", "skip": ["all"], "reason": "need user input"},
-    { "name": "test_http_get.js", "timeout": 20, "skip": ["nuttx"], "reason": "not implemented for nuttx" },
-    { "name": "test_http_header.js", "skip": ["nuttx"], "reason": "not implemented for nuttx" },
-    { "name": "test_httpclient_timeout.js", "timeout": 10, "skip": ["nuttx"], "reason": "not implemented for nuttx" },
-    { "name": "test_httpclient_timeout2.js", "timeout": 15, "skip": ["nuttx"], "reason": "not implemented for nuttx" },
-    { "name": "test_httpserver_timeout.js", "timeout": 10, "skip": ["nuttx"], "reason": "not implemented for nuttx" },
-    { "name": "test_httpserver.js", "timeout": 20, "skip": ["nuttx"], "reason": "not implemented for nuttx" },
+    { "name": "test_fs_event.js", "skip": ["nuttx"], "reason": "not implemented for nuttx" },
+    { "name": "test_fs_open_read_async.js", "skip": ["nuttx"], "reason": "not implemented for nuttx" },
```

Comment:
The `openSync` test below in `test_fs_open_read_async.js` is found. We can move it.

```
// test sync open & read
try {
  var fd = fs.openSync(fileName, flags, mode);
  ...
}
```